### ISSUE TITLE
When i m start google assistant service my pi screen get blank ?

### ISSUE THREAD

**Issue by imrushi** [shivasiddharth/GassistPi#227]
hey i m install google assistant when i m start google assistant service my raspberry pi screen turn in to white screen in vnc is working properly and i m stop that service after reboot the pi then work properly. please solve screen blank error.

**Comment by shivasiddharth** [comment]
The assistant has no dependencies with the display out. If you can provide some images/screenshots it will help give me a better understanding of your issue.

**Comment by imrushi** [comment]
here is screen shot before start service and after stop service.

https://photos.app.goo.gl/4qLeAMnvDQik42FQ2

**Comment by dwh6411** [comment]
I think this is the gpio issue, which is conflict with the gpio used by the display spi display .

**Comment by shivasiddharth** [comment]
@dwh6411 , Yup this a GPIO issue. @imrushi , Please comment the GPIO usages in main.py and actions.py that overlap with your display GPIOs

**Comment by imrushi** [comment]
@shivasiddharth thanks now its working
